Fixes titles for consistency and sentence casing (#19336)
mdn/content: files/en-us/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md

{{ EmbedLiveSample('Controlling_the_order', '230', '460') }}

## Next steps

> In this article, we took a very quick look through the Grid Layout specification. Have a play with the code examples, and then move onto [the next part of this guide](/en-US/docs/Web/CSS/CSS_Grid_Layout/Relationship_of_Grid_Layout), where we will really start to dig into the detail of CSS Grid Layout.

Reviewer: We didn't walk through the spec. We walk through the feature.
```suggestion
In this article, we took a very quick look at the possibilities of grid layouts. Play with the code examples, and then move on to [the next part of this guide](/en-US/docs/Web/CSS/CSS_Grid_Layout/Relationship_of_Grid_Layout), where we will start to dig into the detail of CSS Grid Layout.
```
